### DISCUSSION TITLE
Any A-Z samples?

### DISCUSSION THREAD

**Discussion by 4044ever** [ArturoDent/find-and-transform#6] [Q&A] [answered]
You have any easy A to Z working sample? So far it ain't working for me. 
App is installed.

I have `keybindings.json` in` .vscode` with:

```
{
  "key": "alt+y",
  "command": "findInCurrentFile",
  "args": {
    "find": "(create|table|exists)",
    "replace": "\\U$1",
    "isRegex": true
  }
}
```
I have a sample.md with _create if table exists_ . alt+y has no conflict. But alt+y executed while in sample.md does not trigger anything. Something seems missing, or isn't compatible.

-------------------
OK, I worked it out. You need to:
ctrl+shift+p  >  Preferences: Open Keyboard Shortcuts > click Open Keyboard Shortcuts (JSON) (it's top right)

**Comment by ArturoDent** [comment, marked as answer]
Glad you figured it out.  Yes, `keybindings.json` refers to the one you opened.  

![createTableExistsDemo](https://user-images.githubusercontent.com/31813701/170133676-4c98d7ca-e32c-48a0-bc60-894bda1ba3c5.gif)
